### PR TITLE
bz 1790533: CNV Operator name update for 4.3

### PR DESCRIPTION
**Pull request by ousleyp**
- BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1790533
- For enterprise-4.3 and enterprise-4.4 only
- http://file.bos.redhat.com/pousley/012520/operator-name-2.2/cnv/cnv_install/installing-container-native-virtualization.html
- http://file.bos.redhat.com/pousley/012520/operator-name-2.2/cnv/cnv_install/uninstalling-container-native-virtualization.html
- This PR includes corrections for https://github.com/openshift/openshift-docs/commit/c1f8e368c265887c4eff22d037bf19df281177ae

### CODE REVIEW

**Comment by ousleyp** [comment]
@alexxa Hi Irina, will you please test this for 2.2? I tested it on a 4.3 cluster but I suspect that the version that I have isn't completely up to date for a few reasons:
- @tiraboschi said that the "Operator" part of the name is being removed for 2.2, but it was still there. Let me know if I should add it back in.
- The update channel was still 2.1
- I also am not sure if the name of the CR is supposed to be "kubevirt-hyperconverged" (which is what I saw in the UI) or "hyperconverged-cluster" (how it currently is in this draft)

Let me know if you have any questions or concerns! Thanks :)

**Review by kalexand-rh — COMMENTED** [review]
LGTM

File: modules/cnv-deleting-custom-resource.adoc [hunk 1]
```diff
@@ -19,29 +19,22 @@ the *Projects* list.
...
 . Navigate to the *Operators* -> *Installed Operators* page.
 
-. Click *KubeVirt HyperConverged Cluster Operator*.
+. Click *{CNVProductNameStart}*.
 
 . Click the *KubeVirt HyperConverged Cluster Operator Deployment* tab.
 
-. Click the Options menu {kebab} in the row containing the *hyperconverged-cluster*
-custom resource. In the expanded menu, click *Delete HyperConverged*.
+. Click the Options menu {kebab} in the row containing the *kubevirt-hyperconverged*
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/cnv-deleting-custom-resource.adoc]: @nellyc The custom resource itself is called `kubevirt-hyperconverged` in the most recent screenshot I received from @alexxa (under "Hyper Convergeds")

[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/cnv-deleting-custom-resource.adoc]: From @alexxa: http://pkgs.devel.redhat.com/cgit/containers/hco-bundle-registry/tree/olm-catalog/kubevirt-hyperconverged/2.2.0/kubevirt-hyperconverged-operator.v2.2.0.clusterserviceversion.yaml?h=cnv-2.2-rhel-8#n12

[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/cnv-deleting-custom-resource.adoc]: From @alexxa: according to CNV CSV 2.2 version, line #12, the correct name is `kubevirt-hyperconverged`

**Comment by alexxa** [comment]
@nellyc hyperconverged-cluster comes from the scripts qe (and maybe dev) use. CSV line 12, contains kubevirt-hyperconverged name (doubled checked with Simone today), and when a user opens default yaml to create a CR, they will see kubevirt-hyperconverged. They may change it to whatever they want.
image

**Pull request merged**